lib/stack_master/command.rb

```diff
@@ -36,9 +36,26 @@ def success?
     def error_message(e)
       msg = "#{e.class} #{e.message}"
       msg << "\n Caused by: #{e.cause.class} #{e.cause.message}" if e.cause
+      if defined?(@options)
+        if @options&.trace
+          msg << "\n#{backtrace(e)}"
+        else
+          msg << "\n Use --trace to view backtrace"
```

Comment:
Print the backtrace if the `--trace` option is provided. Otherwise include a message to inform the user to use `--trace` if more details are needed.